ui_framework/components/info_panel/_info_panel.scss

```diff
@@ -1,8 +1,59 @@
 .kuiInfoPanel {
-  padding: 20px;
+  padding: 14px 20px 18px;
   line-height: $lineHeight;
+  border: 2px solid;
+}
+
+/**
+ * 1. TODO: Pick a hex value instead of making these colors translucent.
```

Comment:
It's not really something worth recording in an issue... it's the kind of thing that could possibly never be addressed and we'd be fine. But if we ever encounter a problem with this outline being semi-transparent, I wanted to leave a note that the transparency isn't deliberate, so it's OK to make it opaque. I can remove the `TODO` label and rephrase it if you prefer?